src/test/resources/alma-fix/(DE-605)HT018295975.json

```diff
@@ -74,6 +74,11 @@
     "label" : "Inhaltstext",
```

Comment:
why adding this test file? It looks like src/test/resources/alma-fix/(DE-605)HT015865114.json already covers the test.

Reply:
https://github.com/hbz/lobid-resources/pull/1394/commits/995b9e55dc85247c9108a07e8f0a29b3d727b6f3